case-lib/lib.sh

```diff
@@ -412,3 +412,26 @@ print_module_params()
     grep -H ^ /sys/module/*sof*/parameters/*
     echo "----------------------------------------"
 }
+
+# check-alsabat.sh need to run optimum alsa control settings
+# param1: platform name
+set_alsa_settings()
+{
+    local PNAME="$1"
+    dlogi "alsa setting for $PNAME"
+    case $PNAME in
+        APL_UP2_NOCODEC | APL_UP2_NOCODEC_ZEPHYR | CML_RVP_NOCODEC | JSL_RVP_NOCODEC | JSL_RVP_NOCODEC_ZEPHYR)
```

Comment:
hi @fredoh9  , will TGLU_RVP_NOCODEC_CI use CAVS_NOCDEC.sh here ? I just found 'amixer -c sofnocodec cset name='PGA1.1 1 Master Playback Volume' 32' too high for TGLU_RVP_NOCODEC _CI